docs/recipe_development.rst

```diff
@@ -0,0 +1,132 @@
+==============
+Recipe Development
+==============
+
+mBuild encourages users to subclass ``Compound`` to disseminate reproducible scripts used for building their systems.
+Sharing the script used to construct the initial configuration and apply force field parameters (if applicable) is a great step towards increasing the reproducibility of the system initialization step.
+
+However, there may be cases where those subclasses and/or scripts can be generalized to support a broad range of systems.
+Such subclasses would be a valuable resource for many researchers, and might justify development of a Python package that could be distributed to the community.
+
+
+mBuild has been developed with this in mind, in the form of a plug-in recipe system.
```

Comment:
`plug-in` vs. `recipe`

I think this is some fluid representation we (I) need to nip in the bud. We should pick one of these two and roll with it; as far as I can tell their meaning is identical.

Reply:
What do we want to go with? Currently im using `plug-in` but we can change it later.

Reply:
Typo on my part - I mean `plugin` vs `recipe`